samples/Samples.AspNetCore31/Properties/launchSettings.json

```diff
@@ -19,7 +19,8 @@
         "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
 
         "DD_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
-        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json"
+        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
+        "DD_VERSION": "1.0.0"
```

Comment:
Why is it necessary to add the version to the samples? Testing purposes?

Reply:
Yes, only for testing. I was running the samples manually and looking at the span tags. Not necessary for automated tests. Should I remove them?

Reply:
No, ok to leave them if it helps.